pyproject.toml

```diff
@@ -17,6 +17,10 @@ profile = "black"
 line_length = 100
 [build-system]
 requires = [
+    "requests>=2.24",
+    "toml>=0.10.2",
+    "urllib3>=1.24.2",
+    "pyxdg>=0.26",
```

Comment:
(Just a random thought)

Is there a way I can ingest this file during the VM Image build workflow (in c/automation_images), to check if those requirements are met?

This might be worth doing, since otherwise we only find out about mismatches until after the image build is finished and any fix would require doing it again.

Reply:
I will have to research.

Reply:
There's no reason the c/automation_images CI can't git-clone this repo, and do something really simple, like: use `jq` on the file, feed it to `pip` and `grep -q` for n instances of `not installing...` type messages.